core/modules/layout/css/layout.flexible.admin.css

```diff
@@ -39,14 +39,12 @@
 
 .layout-flexible-editor .flexible-row {
   padding: 0;
-  border: 1.25px dashed #0074bd;
+  border: 1.25px solid #CCC;
```

Comment:
This needs to be lowercase:
`border: 1px solid #ccc;` 
why `1.25`? It looks like 1px on the manage blocks UI?

Reply:
I took the 1.25 px that was there before. Agree that 1px probably makes more sense and is more consistent.